repository/src/main/resources/db/migration/V20211220_12979__RDM-12979__FlagsFlagDetails.sql

```diff
@@ -0,0 +1,117 @@
+(select id from field_type where reference = 'FlagDetails' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('subTypeValue', 'Value', 'PUBLIC',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'FlagDetails' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('subTypeKey', 'Key', 'PUBLIC',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'FlagDetails' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('otherDescription', 'Other Description', 'PUBLIC',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'FlagDetails' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('flagComment', 'Comments', 'PUBLIC',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'FlagDetails' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('dateTimeModified', 'Modified Date', 'PUBLIC',
+(select id from field_type where reference = 'DateTime' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'FlagDetails' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('dateTimeCreated', 'Created Date', 'PUBLIC',
+(select id from field_type where reference = 'DateTime' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'FlagDetails' and version = 1 and jurisdiction_id is null));
+
+
+--- Path and Path Collection ---
+INSERT INTO public.field_type (created_at, reference, base_field_type_id, version)
+VALUES (now(), 'Path',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+'1');
+
+INSERT INTO public.field_type (created_at, reference, base_field_type_id, collection_field_type_id, version)
+VALUES (now(), 'PathCollection',
+(select id from field_type where reference = 'Collection' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'Path' and version = 1 and jurisdiction_id is null), '1');
```

Comment:
Could we just use `Text` here instead of needing to first introduce a new field type named `Path` which is just a copy?

Reply:
Ok